finder-app/finder.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# CU AESD Assignment 1
+# Katie Biggs
+# Jan 21, 2024
+
+# Store off arguments for easier readability
+filesdir=$1
+searchstr=$2
+
+#echo "filesdir: $filesdir"
+#echo "searchstr: $searchstr"
+
+# Return error if incorrect number of parameters specified
+if [ $# != 2 ]; then
+	echo "Incorrect number of parameters. Expecting 2."
```

Comment:
@kbiggs I like the idea to put an error message here but how could you make it more useful for the user?